### ISSUE TITLE
Do you think that there will be a version for Visual Studio 2022?

### ISSUE THREAD

**Issue by ragininternet** [Art-Stea1th/Enhanced-Syntax-Highlighting#26]
Thanks for your work. Love the tool.

Do you think that there will be a version for Visual Studio 2022?

**Comment by Trevor89** [comment]
Agreed, I updated to 2022 and I miss the syntax highlighting I have become accustomed to. 
Also your ESH works great on C/C++ so you should add that to your front page readme next to C# and VB.

Trev

**Comment by Trevor89** [comment]
I edited the manifest (renamed vsix to zip) and got it to install and it works.
If this isnt the solution (and my installation actually scilently failed?) then it would seen that 2022 already has enhanced highlighting (but you need to specify different colours)
Anyway, the "user tags" that ESH places into "Fonts and colours" do show up so I think it did install correctly.
Trev

**Comment by MihaMarkic** [comment]
@Trevor89 @ragininternet  Meanwhile you can try my fork https://github.com/MihaMarkic/Enhanced-Syntax-Highlighting/releases/tag/1.2.2022
I've updated references and bumped .NET Framework to 4.7.2
"It works on my machine"
